ansible_events_ui/api/activation.py

```diff
@@ -45,3 +45,93 @@ async def create_activation(
     (id_,) = result.inserted_primary_key
 
     return {**activation.dict(), "id": id_}
+
+
+@router.get(
+    "/api/activation/{activation_id}",
+    response_model=schemas.ActivationRead,
+    operation_id="show_activation",
+)
+async def read_activation(
+    activation_id: int, db: AsyncSession = Depends(get_db_session)
+):
+    query = (
+        sa.select(
+            models.activations.c.id,
+            models.activations.c.name,
+            models.activations.c.description,
+            models.activations.c.activation_enabled,
+            models.activations.c.activation_status,
+            models.activations.c.restarted_at,
+            models.activations.c.restarted_count,
+            models.activations.c.created_at,
+            models.activations.c.modified_at,
+            models.rulebooks.c.id.label("rulebook_id"),
```

Comment:
The response includes a lot of related entities and their fields. I think some level of nesting may be nice to have, it's better aligned with ORM relationship configuration once we switch there, but on the other hand it will require more complicated schemas. For example instead of

```
{
    "id": 1,
    "name": "somename",
    "rulebook_id": 1,
    "rulebook_name": "testrulebook.yaml"
    "inventory_id": 1,
    "inventory_name: "testinventory.yaml"
    ...
}
```
We can have something like:
```
{
    "id": 1,
    "name": "somename",
    "rulebook: { "id": 1, "name": "testrulebook.yaml" },
    "inventory": { "id": 1, "name: "testinventory.yaml" },
    ...
}
```

@lgalis which one is more preferable for you: flat or nested?